src/legacy/core_plugins/data/public/search/search_bar/components/search_bar.tsx

```diff
@@ -183,10 +185,15 @@ class SearchBarUI extends Component<SearchBarProps, State> {
   }
 
   public render() {
+    if (!this.props.uiSettings) {
+      return null;
+    }
```

Comment:
Isn't `uiSettings` required? Why was this needed?

Reply:
Removed, suspect it might make a test fail, but lets see!

Reply:
It's needed because the kbn-top-nav-v2 directive might load it before npSetup was initialized.
I added a comment there.